modular_skyrat/modules/loadouts/loadout_items/loadout_datum_toys.dm

```diff
@@ -178,6 +178,22 @@ GLOBAL_LIST_INIT(loadout_toys, generate_loadout_items(/datum/loadout_item/toys))
 /datum/loadout_item/toys/lizzyplush
 	name = "Odd yoga lizzy plushie"
 	item_path = /obj/item/toy/plush/lizzyplush
+	
+/datum/loadout_item/toys/bluecat
+	name = "Blue Cat Plushie"
+	item_path = /obj/item/toy/plush/bluecat
+
+/datum/loadout_item/toys/igneoussynth
+	name = "Igneous Synth Plushie"
+	item_path = /obj/item/toy/plush/igneoussynth
+
+/datum/loadout_item/toys/edgybird
+	name = "Edgy Bird Plushie"
+	item_path = /obj/item/toy/plush/edgybird
+
+/datum/loadout_item/toys/treeferret
+	name = "Tree Ferret Plushie"
+	item_path = /obj/item/toy/plush/treeferret
 
```

Comment:
I know we're looser in terms of coding standards for donator stuff, but it still bothers me, let's move this to snake_case please.
```suggestion
/datum/loadout_item/toys/blue_cat
	name = "Blue Cat Plushie"
	item_path = /obj/item/toy/plush/blue_cat

/datum/loadout_item/toys/igneous_synth
	name = "Igneous Synth Plushie"
	item_path = /obj/item/toy/plush/igneous_synth

/datum/loadout_item/toys/edgy_bird
	name = "Edgy Bird Plushie"
	item_path = /obj/item/toy/plush/edgy_bird

/datum/loadout_item/toys/tree_ferret
	name = "Tree Ferret Plushie"
	item_path = /obj/item/toy/plush/tree_ferret

```